grakn-graph/src/main/java/ai/grakn/graph/internal/RelationTypeImpl.java

```diff
@@ -43,14 +42,22 @@
  *
  */
 class RelationTypeImpl extends TypeImpl<RelationType, Relation> implements RelationType {
-    RelationTypeImpl(AbstractGraknGraph graknGraph, Vertex v, Optional<RelationType> type, Optional<Boolean> isImplicit) {
+    RelationTypeImpl(AbstractGraknGraph graknGraph, Vertex v) {
+        super(graknGraph, v);
+    }
+
+    RelationTypeImpl(AbstractGraknGraph graknGraph, Vertex v, RelationType type) {
+        super(graknGraph, v, type);
+    }
```

Comment:
If we get rid of the `if` statements above, maybe we can get rid of this constructor.